qmxgraph/page/api.js

```diff
@@ -1052,6 +1052,7 @@ graphs.Api.prototype.setStyle = function setStyle(cellId, style) {
     var model = graph.getModel();
     var cell = this._findCell(model, cellId);
     model.setStyle(cell, style);  // Use the model to trigger mxStyleChange as needed.
+    graph.refresh(cell); // Force a refresh of that cell to guarantee
```

Comment:
@prusse-martin any tips on how to test this to avoid future regressions? 🤔

Reply:
Call the api method and check that the rendered cell have been updated.
Some tests already do something similar:
- test_container_resize in [get_container_dimensions](https://github.com/ESSS/qmxgraph/blob/master/tests/test_qt_js_integration.py#L177-L184);
- [test_custom_shapes](https://github.com/ESSS/qmxgraph/blob/master/tests/test_js_graph.py#L630);
- [test_edge_with_style](https://github.com/ESSS/qmxgraph/blob/master/tests/test_js_graph.py#L688);
- [test_custom_font_family](https://github.com/ESSS/qmxgraph/blob/master/tests/test_js_graph.py#L1431);

I would pretty much like to see a test in the `test_js_graph.py` style if the issue does not only affect qt based rendering.
But both `test_js_graph.py` and `test_qt_js_integration.py`.

Reply:
Maybe I can have a try on this later if you guys arenot in a runch.

Reply:
I tried to craft a test but them pointed that this is not required since the test passed without this change.

Reply:
Yep, I don't know how to make this happen usingn qmxgraph only, maybe we need to investigate further in ALFAsim to detect the necessary context for this misbehavior.